python/rpm/__init__.py

```diff
@@ -126,3 +126,24 @@ def dsSingle(TagN, N, EVR="", Flags=RPMSENSE_ANY):
     dsSingle(RPMTAG_CONFLICTNAME, "rpm") corresponds to "Conflicts: rpm"
     """
     return ds((N, Flags, EVR), TagN)
+
+def parse_evr(evr_string):
```

Comment:
note that there is same function in C and I think we should really expose that one.